docker-compose-runner.yml

```diff
@@ -34,3 +34,10 @@ services:
       - "5432:5432"
     environment:
       - POSTGRES_HOST_AUTH_METHOD=trust
+  postgres_tmp:
```

Comment:
Maybe something like `postgres_for_unit_tests` would be better? Esp with a comment like "An instance of postgres configured to execute Rust unit-tests, tuned for performance".

Reply:
done